There is no reason to run this script and running it fails on architectures
chromium does not officially support (such as ppc64).

--- qtwebengine/src/3rdparty/chromium/build/config/posix/BUILD.gn
+++ qtwebengine/src/3rdparty/chromium/build/config/posix/BUILD.gn
@@ -102,12 +102,12 @@ config("runtime_library") {
       # when turning the sysroot on or off. (defines are passed via the command
       # line, and build system rebuilds things when their commandline
       # changes). Nothing should ever read this define.
-      sysroot_hash =
-          exec_script("//build/linux/sysroot_scripts/install-sysroot.py",
-                      [ "--print-hash=$current_cpu" ],
-                      "trim string",
-                      [ "//build/linux/sysroot_scripts/sysroots.json" ])
-      defines += [ "CR_SYSROOT_HASH=$sysroot_hash" ]
+      #sysroot_hash =
+      #    exec_script("//build/linux/sysroot_scripts/install-sysroot.py",
+      #                [ "--print-hash=$current_cpu" ],
+      #                "trim string",
+      #                [ "//build/linux/sysroot_scripts/sysroots.json" ])
+      #defines += [ "CR_SYSROOT_HASH=$sysroot_hash" ]
     }
     asmflags += sysroot_flags
 
